pmm-app/tests/verifyQAN_test.js

```diff
@@ -121,3 +122,99 @@ Scenario('Verify Main Metric change reflects in URL @not-pr-pipeline', async (I,
   qanPage.changeMetricTo(metricToReplace, newMetricName);
   qanPage.verifyURLContains(qanPage.urlParts.lockTime);
 });
+
+xScenario(
```

Comment:
@IvanaWzientkova Since we don't run these tests, can we commit them when they are ready to be run?